examples/presenting_revoked_credential_anoncreds/example.py

```diff
@@ -0,0 +1,186 @@
+            await bob.record(topic="revocation-notification")
+            revoked_time = int(time.time())
+
+        # Request proof from holder again after revoking,
+        # using the interval before cred revoked
+        await indy_present_proof_v2(
+            bob,
+            alice,
+            bob_conn.connection_id,
+            alice_conn.connection_id,
+            requested_attributes=[
+                {
+                    "name": "firstname",
+                    "restrictions": [
+                        {"cred_def_id": cred_def["credential_definition_id"],}
+                    ],
+                }
+            ],
+            non_revoked={"from": issued_time, "to": issued_time},
+        )
+
+        # Request proof, no interval
+        await indy_present_proof_v2(
+            bob,
+            alice,
+            bob_conn.connection_id,
+            alice_conn.connection_id,
+            requested_attributes=[
+                {
+                    "name": "firstname",
+                    "restrictions": [
+                        {"cred_def_id": cred_def["credential_definition_id"],}
+                    ],
+                }
+            ],
+        )
+
+        # Request proof, using invalid/revoked interval but using
+        # local non_revoked override (in requsted attrs)
+        # ("LOCAL"-->requested attrs)
+        await indy_present_proof_v2(
+            bob,
+            alice,
+            bob_conn.connection_id,
+            alice_conn.connection_id,
+            requested_attributes=[
+                {
+                    "name": "firstname",
+                    "restrictions": [
+                        {"cred_def_id": cred_def["credential_definition_id"],}
+                    ],
+                    "non_revoked": {
+                        "from": issued_time,
+                        "to": issued_time,
+                    },
+                }
+            ],
+            non_revoked={"from": revoked_time - 1, "to": revoked_time},
+        )
+
+        # Request proof, just local invalid interval
+        await indy_present_proof_v2(
+            bob,
+            alice,
+            bob_conn.connection_id,
+            alice_conn.connection_id,
+            requested_attributes=[
+                {
+                    "name": "firstname",
+                    "restrictions": [
+                        {"cred_def_id": cred_def["credential_definition_id"],}
+                    ],
+                    "non_revoked": {
+                        "from": revoked_time,
+                        "to": revoked_time,
+                    },
+                }
+            ],
+        )
+
+        # Query presentations
+        presentations = await alice.get(
+            "/present-proof-2.0/records",
+            response=V20PresExRecordList,
+        )
+
+        # Presentation summary
+        for i, pres in enumerate(presentations.results):
+            print(summary(pres))
```

Comment:
We can remove these lines from this example; since the main goal of this one is to exercise the anoncreds endpoints, we can let the `presenting_revoked_credentials` example exercise ACA-Py's revocation behavior.